From c02bb7dce587886fe380704a8c6a6009eb677a74 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Sat, 21 May 2022 15:41:11 +0200
Subject: [PATCH] Fixed duplicate subparser name in
 test_add_parser_custom_completer

If I'm not mistaken, the use of the same name for both subparsers was
not intentional but a typo.  In Python 3.11, this is an error and causes
the test to fail.

Fixes #1228
---
 tests/test_argparse_completer.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test_argparse_completer.py b/tests/test_argparse_completer.py
index 135d3de4..14c8c1de 100644
--- a/tests/test_argparse_completer.py
+++ b/tests/test_argparse_completer.py
@@ -1371,5 +1371,5 @@ def test_add_parser_custom_completer():
     no_custom_completer_parser = subparsers.add_parser(name="no_custom_completer")
     assert no_custom_completer_parser.get_ap_completer_type() is None  # type: ignore[attr-defined]
 
-    custom_completer_parser = subparsers.add_parser(name="no_custom_completer", ap_completer_type=CustomCompleter)
+    custom_completer_parser = subparsers.add_parser(name="custom_completer", ap_completer_type=CustomCompleter)
     assert custom_completer_parser.get_ap_completer_type() is CustomCompleter  # type: ignore[attr-defined]
